src/guides/v2.4/rest/generate-local.md

```diff
@@ -60,7 +60,25 @@ To return the complete JSON schema, specify the `?services=all` parameter in the
 
 The base URL for returning the asynchronous schema is `http://<magento_host>/rest/<store_code>/async/schema`.
 
+## Security
+
+By default, the `/swagger` endpoint is open on Magneto instances. Leaving it open on a public instance could potentially expose system information that should not be readable. 
```

Comment:
```suggestion
By default, the `/swagger` endpoint is open on Magento instances. Leaving it open on a public instance could potentially expose system information that should not be readable. 
```

Reply:
Of all the words to misspell, it's the name of my employer.